lib/Backend/FlowGraph.cpp

```diff
@@ -95,13 +248,109 @@ FlowGraph::Build(void)
             }
             break;
 
+        case Js::OpCode::Finally:
+        {
+            if (!this->finallyLabelStack)
+            {
+                break;
+            }
+
+            //To enable globopt on functions with try finallys we transform the flowgraph as below :
+            //          TryFinally L1
+            //          <try code>
+            //          Leave L2
+            //  L2 :    Br L3
+            //  L1 :    <finally code>
+            //          LeaveNull
+            //  L3 :    <code after try finally>
+            //
+            //to:
+            //
+            //          TryFinally L1
+            //          <try code>
+            //          BrOnException L1
+            //          Leave L1'
```

Comment:
It appears to me that L1' is really L2 from before the transformation. If so, lets keep L2 only after the transformation. Makes understanding the code below easier